includes/woopay/class-woopay-session.php

```diff
@@ -902,6 +902,7 @@ private static function get_option_fields_status() {
 		$address_2      = get_option( 'woocommerce_checkout_address_2_field', 'optional' );
 		$phone          = get_option( 'woocommerce_checkout_phone_field', 'required' );
 		$terms_checkbox = ! empty( get_option( 'woocommerce_terms_page_id', null ) );
```

Comment:
> Would it be better to use [wc_terms_and_conditions_checkbox_enabled()](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/wc-template-functions.php?rgh-link-date=2024-12-10T20%3A30%3A24Z#L835-L845)?

> We have a discrepancy rendering of the T&C checkbox between the merchant store and WooPay with the current approach:


@pierorocca What should WooPay behave in case there is a custom Terms and conditions text on /wp-admin > Payments > WooPay > Terms and Conditions, there is a terms and conditions page set on the shortcode checkout settings but there is no shortcode terms and conditions?